### DISCUSSION TITLE
Opening video from Youtube URL is not working

### DISCUSSION THREAD

**Discussion by 2bear1124** [SubtitleEdit/subtitleedit#7243] [Q&A] [answered]
Hello. Thanks for making this program.

My problem: I was trying to open a video with URL from Youtube but it is not working. (I cannot play a video on the program)
Can you please help me to fix this problem?
(I'm using the latest version of this program and Window 10) 
Also I even tried to reinstall this program, and checked if mpv lib and youtube-dl were installed well.

**Comment by niksedk** [comment, marked as answer]
Could you try latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip
I've just updated yt-dlp to [2023.07.06](https://github.com/yt-dlp/yt-dlp/releases/tag/2023.07.06)

**Reply by 2bear1124** [reply]
It's working !!!!!!!! 
Thank you so much for your kindness!!!
